app/controllers/dashboard_controller.rb

```diff
@@ -779,7 +779,6 @@ def save_user_dashboards
 
   # Gather information for the report accordians
   def build_timeline_listnav
-    populate_reports_menu("timeline", "menu")
```

Comment:
This will break Gaprindashvili. Fixed https://github.com/ManageIQ/manageiq-ui-classic/pull/3476 .